.buildkite/build-test-omnibus.sh

```diff
@@ -21,92 +26,262 @@ for platform in ${platforms[@]}; do
-    echo "      privileged: true"
-    echo "      propagate-environment: true"
-    echo "      environment:"
-    echo "        - RPM_SIGNING_KEY"
-    echo "        - CHEF_FOUNDATION_VERSION"
-    echo "  commands:"
-    echo "    - ./.expeditor/scripts/omnibus_chef_build.sh"
-    echo "  timeout_in_minutes: 60"
-  else 
-    echo "- label: \":hammer_and_wrench::windows: $platform\""
-    echo "  retry:"
-    echo "    automatic:"
-    echo "      limit: 1"
-    echo "  key: build-$platform"
+    echo "  - chef/omnibus#852c8f81fb6dd12ff3471a8d825ec20a1168c4c4:"
+    echo "      build: chef"
+    echo "      chef-foundation-version: $CHEF_FOUNDATION_VERSION"
+    echo "      config: omnibus/omnibus.rb"
+    echo "      install-dir: \"/opt/chef\""
+    if [ $build_key == "mac_os_x-10_15-x86_64" ]
+    then
+      echo "      remote-host: buildkite-omnibus-$platform"
+    fi
+    echo "      omnibus-pipeline-definition-path: \".expeditor/release.omnibus.yml\""
+    if [ $build_key == "mac_os_x-11-arm64" ]
+    then
+      echo "  concurrency: 1"
+      echo "  concurrency_group: omnibus-$build_key/build/chef"
+    fi
+  done
+
+  if  [[ " ${esoteric_build_platforms[*]} " =~ "mac_os_x" ]]
+  then
+    echo "- key: notarize-macos"
+    echo "  label: \":lock_with_ink_pen: Notarize macOS Packages\""
     echo "  agents:"
-    echo "    queue: default-$platform-privileged"
+    echo "    queue: omnibus-mac_os_x-12-x86_64"
     echo "  plugins:"
-    echo "  - docker#v3.5.0:"
-    echo "      image: chefes/omnibus-toolchain-$platform:$OMNIBUS_TOOLCHAIN_VERSION"
-    echo "      shell:"
-    echo "      - powershell"
-    echo "      - \"-Command\""
-    echo "      propagate-environment: true"
-    echo "      environment:"
-    echo "        - CHEF_FOUNDATION_VERSION"
-    echo "        - BUILDKITE_AGENT_ACCESS_TOKEN"
-    echo "        - AWS_ACCESS_KEY_ID"
-    echo "        - AWS_SECRET_ACCESS_KEY"
-    echo "        - AWS_SESSION_TOKEN"
-    echo "      volumes:"
-    echo '        - "c:\\buildkite-agent:c:\\buildkite-agent"'
-    echo "  commands:"
-    echo "    - ./.expeditor/scripts/omnibus_chef_build.ps1"
-    echo "  timeout_in_minutes: 120"
+    echo "  - chef/omnibus#v0.2.81:"
+    echo "      config: omnibus/omnibus.rb"
+    echo "      remote-host: buildkite-omnibus-mac_os_x-12-x86_64"
+    echo "      notarize-macos-package: chef"
+    echo "      omnibus-pipeline-definition-path: \".expeditor/release.omnibus.yml\""
+    echo "  depends_on:"
+    for platform in ${esoteric_build_platforms[@]}; do
+      if [[ $platform =~ mac_os_x ]]
+      then
+        echo "  - $platform"
+      fi
+    done
   fi
-done
+fi
+
+if [ $BUILDKITE_PIPELINE_SLUG == "chef-chef-main-validate-release" ]
+then
+  echo "- wait: ~"
+  echo "- key: create-build-record"
+  echo "  label: \":artifactory: Create Build Record\""
+  echo "  plugins:"
+  echo "  - chef/omnibus#v0.2.83:"
```

Comment:
We need to align these versions throughout. We have some using 0.2.81 and others using 0.2.83